ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -48,19 +50,7 @@ PARTITION_DATABASE_CONFIG = {
 }
 {% endif %}
 
-{% if localsettings.get('UCR_DATABASE_NAME') %}
-UCR_DATABASE_URL = "postgresql://{{ localsettings.PG_DATABASE_USER }}:{{ localsettings.PG_DATABASE_PASSWORD }}@{{ localsettings.PG_DATABASE_HOST }}:{{localsettings.get('PG_DATABASE_PORT')}}/{{ localsettings.UCR_DATABASE_NAME }}"
-{% endif %}
-
-{% if localsettings.get('ICDS_UCR_DATABASE_ALIAS') %}
-ICDS_UCR_DATABASE_ALIAS = "{{ localsettings.ICDS_UCR_DATABASE_ALIAS }}"
-{% endif %}
-
-{% if localsettings.get('ICDS_UCR_TEST_DATABASE_ALIAS') %}
-ICDS_UCR_TEST_DATABASE_ALIAS = "{{ localsettings.ICDS_UCR_TEST_DATABASE_ALIAS }}"
```

Comment:
@snopoke not having htis is causing dashboard errors https://sentry.io/dimagi/commcarehq/issues/337569108/?referrer=slack. Is it due to not having a new code deploy?

Reply:
sorry about that, thanks for the fix